Projects/Application/Sources/Flow/MainFlow/FeedScreen/FeedViewReactor.swift

```diff
@@ -57,7 +60,16 @@ final class FeedViewReactor: Reactor {
             let finishLoading = Observable.just(Mutation.setLoadingStatus(.none))
             return .concat(startLoading, fetchPosts, finishLoading)
         case .loadNextPage:
-            return .empty()
+            guard currentState.loadingStatus == .none else { return  .empty() }
+
+            let startLoading = Observable.just(Mutation.setLoadingStatus(.loadingNextPage))
+
+            let fetchPosts = dependencies.subredditService
+                .getPosts(for: "popular")
+                .map { Mutation.appendPosts($0) }
```

Comment:
From my understanding, this change allows our app to continuously load posts; if so, shouldn't there be a pagination system in place?

Reply:
Service is not ready for that yet, this PR only about the visual part.